src/libraries/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -732,13 +734,13 @@ public partial class JsonConverterAttribute : System.Text.Json.Serialization.Jso
     {
         protected JsonConverterAttribute() { }
         public JsonConverterAttribute(System.Type converterType) { }
-        public System.Type ConverterType { get { throw null; } }
-        public virtual System.Text.Json.Serialization.JsonConverter CreateConverter(System.Type typeToConvert) { throw null; }
+        public System.Type? ConverterType { get { throw null; } }
+        public virtual System.Text.Json.Serialization.JsonConverter? CreateConverter(System.Type typeToConvert) { throw null; }
     }
     public abstract partial class JsonConverterFactory : System.Text.Json.Serialization.JsonConverter
     {
         protected JsonConverterFactory() { }
-        public abstract System.Text.Json.Serialization.JsonConverter CreateConverter(System.Type typeToConvert, System.Text.Json.JsonSerializerOptions options);
+        public abstract System.Text.Json.Serialization.JsonConverter CreateConverter(System.Type typeToConvert, System.Text.Json.JsonSerializerOptions? options);
```

Comment:
This shouldn't be nullable.
```suggestion
        public abstract System.Text.Json.Serialization.JsonConverter CreateConverter(System.Type typeToConvert, System.Text.Json.JsonSerializerOptions options);
```